test/Middleware/ImplicitHeadMiddlewareTest.php

```diff
@@ -127,9 +129,9 @@ public function testReturnsComposedResponseWhenPresentWhenRouteImplicitlySupport
     {
         $route = $this->prophesize(Route::class);
         $route->implicitHead()->willReturn(true);
-        $route->allowsMethod(RequestMethod::METHOD_GET)->willReturn(false);
 
         $result = $this->prophesize(RouteResult::class);
+        $result->getAllowedMethods()->willReturn([RequestMethod::METHOD_HEAD]);
```

Comment:
I just realized this one is incorrect; we should likely be using POST or PATCH here; anything BUT HEAD or GET.

Reply:
Hm... It's fine, because this is case when we have implicit head `true`

Reply:
Yeah, just came to that same conclusion. :+1: